components/tooltip/theme.css

```diff
@@ -0,0 +1,229 @@
+@import '@teamleader/ui-colors';
+@import '@teamleader/ui-utilities';
+
+:root {
+  --unit: 10px;
```

Comment:
why not just use the one form `variables.css`?

Reply:
I'd suggest putting **--unit: 10px;** inside the ui-utilities and use it from there?

Reply:
good for me. Do it in a later PR.